test/e2e/apps/statefulset.go

```diff
@@ -843,6 +847,52 @@ var _ = SIGDescribe("StatefulSet", func() {
 				return nil
 			}, framework.StatefulPodTimeout, 2*time.Second).Should(BeNil())
 		})
+
+		It("should have a working scale subresource", func() {
+			By("Creating statefulset " + ssName + " in namespace " + ns)
+			ss := framework.NewStatefulSet(ssName, ns, headlessSvcName, 1, nil, nil, labels)
+			sst := framework.NewStatefulSetTester(c)
+			sst.SetHttpProbe(ss)
+			ss, err := c.AppsV1beta1().StatefulSets(ns).Create(ss)
+			Expect(err).NotTo(HaveOccurred())
+			sst.WaitForRunningAndReady(*ss.Spec.Replicas, ss)
+			ss = sst.WaitForStatus(ss)
+
+			By("posting a scale subresource")
```

Comment:
What is the benefit of this step in the test? It seems to test the same thing as "updating a scale subresource".